types/braintree-web/google-payment.d.ts

```diff
@@ -31,7 +35,40 @@ export interface GooglePaymentTokenizePayload {
     };
 }
 
-export interface GooglePayment {
+type GooglePaymentClientType = "sync" | "deferred";
```

Comment:
As far as I can tell, this is an indirection to make it easier to type

`GooglePaymentBase<google.payments.api.PaymentDataRequest>`
and
`GooglePaymentBase<Promise<google.payments.api.PaymentDataRequest>>`

I would much prefer writing those two instead of "sync" and "deferred".

Reply:
That's right. I did this on purpose since it felt weird to let `google.payments.api.PaymentDataRequest` define the whole interface, when it is actually relevant only to the one of its methods.

But I guess you are right, since the whole interface actually revolves around that one method 👍